### DISCUSSION TITLE
环境安装过程中，报出与c include 路径相关的错误？是否遇到过？

### DISCUSSION THREAD

**Discussion by tpanda09** [chatchat-space/Langchain-Chatchat#677] [Q&A]
在macbook pro m1上安装环境，新建了一个独立python环境，运行pip install -r requirements过程中报错：
Building wheels for collected packages: pymupdf, onnx
Building wheel for pymupdf (setup.py) ... error
error: subprocess-exited-with-error
具体报错信息是： In file included from /Library/Developer/CommandLineTools/usr/include/c++/v1/cstring:60:
/Library/Developer/CommandLineTools/usr/include/c++/v1/string.h:60:15: fatal error: 'string.h' file not found
#include_next <string.h>
^~~~~~~~~~
1 error generated.
简单讲，就是在路径/Library/Developer/CommandLineTools/usr/include/c++/v1/下找不到string.h，但也确实没有这个路径，我的机器设置
CPLUS_INCLUDE_PATH=/Library/Developer/CommandLineTools/usr/include/c++/v1/
C_INCLUDE_PATH=/Library/Developer/CommandLineTools/SDKs/MacOSX13.3.sdk/usr/include
不知系统是怎么拼接的路径。

**Comment by tpanda09** [comment]
随后，做了问题定位尝试
1.注释掉requirement.txt 中的paddleocr~=2.6.1.3
2.执行pip install -r requirement.txt成功
3.独立执行pip install paddleocr==2.6.1.3失败，报出错误和昨天一致
看来和paddleocr有关

**Comment by tpanda09** [comment]
已解决，目前安装不再报错

**Reply by WaterDJiang** [reply]
我也是安装paddleocr 出错，请问怎么解决的？
